packages/@react-aria/grid/src/useGrid.ts

```diff
@@ -73,6 +73,17 @@ export function useGrid<T>(props: GridProps, state: GridState<T, GridCollection<
     console.warn('An aria-label or aria-labelledby prop is required for accessibility.');
   }
 
+  // track keyboard navigation for blocking announcements in "replace" mode
```

Comment:
// thought to wrap all delegate functions such that any keyboard events it handled would set this to true, but decorating an
  // instance seems hard, there are plenty of potential functions that might falsely do something if i just wrap the whole thing
  // could create my own onKeyDown/onKeyUp handler that matches what keys useSelectableCollection does, but that's brittle if "wasd" were to be used
  // could pass an onKeyHandled to useSelectableCollection, but it's a possibly break in API